noxfiles/utils_nox.py

```diff
@@ -69,8 +69,13 @@ def init_saas_connector(session: nox.Session) -> None:
     variable_map = {"connector_name": connector_name, "connector_id": connector_id}
 
     # create empty config and dataset files
-    open(f"data/saas/config/{variable_map['connector_id']}_config.yml", "w")
-    open(f"data/saas/dataset/{variable_map['connector_id']}_dataset.yml", "w")
```

Comment:
nice! looks good.

but...apologies for the nit - but can we do something similar in L91 and L98 though before writing _those_ files? similar concept/concern

Reply:
looks good on latest!